### DISCUSSION TITLE
Import/Export help

### DISCUSSION THREAD

**Discussion by scottiemc7** [OrchardCMS/OrchardCore#11776] [Q&A] [answered]
Hi,

Using the Coming Soon theme, I made some small changes to the main ComingSoon content item locally, and I'm trying to export those changes to my hosting server. After I export all of what I think I need, I'm no longer able to set the 'Set as Homepage' option for the ComingSoon page. The option just never sticks. Does anyone have any idea what would cause that?

Steps to reproduce:

1. Get the latest Orchard version up and running, for me that was `docker run --name orchardcms orchardproject/orchardcore-cms-linux:latest -p 8080:80`
2. Setup the default tenant using the ComingSoon recipe. 
3. Make a change to the ComingSoon content item (or don't, not required). 
4. Export everything using a deployment plan:
  a. All Features
  b. All Templates
  c. All Workflow types
  d. Media
  e. All Content
  f. Update Content Definitions (all)
5. Create a new tenant. Set up using the Blank Site recipe (also tried the Coming Soon recipe).
6. Import the package.
7. Try to set the ComingSoon content item to be the home page. The option never seems to save.

Here's the export package [All(6).zip](https://github.com/OrchardCMS/OrchardCore/files/8790064/All.6.zip)

**Comment by lampersky** [comment, marked as answer]
@scottiemc7 have you solved this issue? if not, here is what I found:

I'm not sure how you did this, but for some reason **AutoroutePart.Path** is **null** for both of your **ComingSoon** items.
Because you have two content items with the same **AutoroutePart.Path** value, you are not able to update it.

I was able to fix this by preparing another recipe with proper **Path** value. You could also fix this via DB viewer.

![image](https://user-images.githubusercontent.com/6403130/179363600-839a6a26-5be9-4569-a333-1daa377a2433.png)

**Reply by ns8482e** [reply]
Is the missing path reproducible via above steps?

**Reply by lampersky** [reply]
Yes, it is

**Reply by scottiemc7** [reply]
@lampersky oh I see, thank you very much. I'm also not sure how exactly this happens, but I don't do anything beyond the steps I've outlined above. It sounds like once I've imported package there's no way to fix it, is that right?

**Reply by lampersky** [reply]
@scottiemc7 

as mentioned before, I was able to repro your issue by creating new site using **ComingSoon** recipe and then importing your recipe.
This results in two different content items (ContentType: ComingSoon) with **null** path for AutoroutePart.

There are two ways how to fix it:
- first described earlier, by creating new recipe file which will fix null paths,
- second, easier but requires few steps:
  - modify your **ComingSoon** ContentType by specifying autoroute pattern ```{{ ContentItem.DisplayText | slugify }}```
![image](https://user-images.githubusercontent.com/6403130/189680615-9b9446cf-dc9b-43f5-a237-b6087d0306fa.png)
  - update all **ComingSoon** content items manually, so their paths are refreshed when saved based on pattern,
  - now you can switch "home page" back and forth between content items.

Hope, that's clear.

IMO the issue here is with **comingsoon.recipe.json**, because there is no pattern defined for **autoroute part** and this results with **null** for every new item created.
